src/query/functions-v2/src/scalars/control.rs

```diff
@@ -140,4 +147,78 @@ pub fn register(registry: &mut FunctionRegistry) {
             }),
         }))
     });
+    registry.register_function_factory(IS_NULL, |_, args_type| {
+        if args_type.len() != 1 {
+            return None;
+        }
+        Some(Arc::new(Function {
+            signature: FunctionSignature {
+                name: IS_NULL,
+                args_type: args_type.to_vec(),
+                return_type: DataType::Boolean,
+                property: FunctionProperty::default(),
+            },
+            calc_domain: Box::new(|args_domain, _| {
+                assert_eq!(args_domain.len(), 1);
+                None
+            }),
+            eval: Box::new(|args, _generics| {
+                assert_eq!(args.len(), 1);
+                match &args[0] {
+                    ValueRef::Column(Column::Nullable(box NullableColumn { validity, .. })) => {
+                        let bitmap = Bitmap::from_iter(validity.iter().map(|v| !v));
+                        Ok(Value::Column(Column::Boolean(bitmap)))
+                    }
+                    ValueRef::Column(column) => {
+                        let len = column.len();
+                        let mut bitmap = MutableBitmap::with_capacity(len);
+                        for _ in 0..len {
+                            unsafe { bitmap.push_unchecked(false) };
+                        }
+                        let bitmap = Bitmap::from(bitmap);
+                        Ok(Value::Column(Column::Boolean(bitmap)))
+                    }
+                    ValueRef::Scalar(ScalarRef::Null) => Ok(Value::Scalar(Scalar::Boolean(true))),
+                    ValueRef::Scalar(_) => Ok(Value::Scalar(Scalar::Boolean(false))),
+                }
+            }),
+        }))
+    });
+    registry.register_function_factory(IS_NOT_NULL, |_, args_type| {
```

Comment:
Maybe we can desugar `is_not_null(x)`to `not(is_null(x))` so that `is_not_null` is not necessarily defined here.

Reply:
Ok, bug how to implement desugar?

Reply:
it'll soon be implemented. we can just remove is_not_null() so far.

Reply:
Sorry, my mistake. We may keep `is_not_null` and discard `is_null` because, in Nullable, validity which is exactly the `is_not_null` is stored.